@css/main_style.css

```diff
@@ -716,8 +713,11 @@ dl.tech-menu > dt {
 }
 
 .form-email-layout {
-  min-width: 327px;
-  height: 48px;
+  display: flex;
+  min-width: 273px;
+  height: 18px;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] you did a great a jib on implementing the contact form  great job but according to the project requirement you must stick to the design template and also The website should be responsive, following these breakpoints:
Minimum mobile screen size should be 375px
Minimum desktop screen size should be 768px

- [ ] see the screeners attached below to see

https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/92458236/b26f118c-63eb-4b05-ae73-47666769ada2